src/index.html

```diff
@@ -163,66 +163,127 @@ <h1>Scaling <i>beyond</i> geospatial tech</h1>
+                <p>
+                  Raster Vision supports customizable Albumentations transforms to use during training.
+                </p>
+              </div>
+              <div class="tech-spec text-white">
+                <img src="images/diagram-sankey.svg" class="h-7 w-7"/>
+                <p class="font-semibold text-med">
+                  Custom model specification
+                </p>
+                <p>
+                  Utilize a wide variety of compatible models from local directories using TorchHub.
+                </p>
+              </div>
+              <div class="tech-spec text-white">
+                <img src="images/flame.svg" class="h-7 w-7"/>
+                <p class="font-semibold text-med">
+                  Based on PyTorch
+                </p>
+                <p>
+                  Raster Vision implements deep learning functionality with PyTorch (a popular deep learning library) and Torchvision.
+                </p>
+              </div>
+              <div class="tech-spec text-white">
+                <img src="images/cloud-word.svg" class="h-7 w-7"/>
+                <p class="font-semibold text-med">
+                  Supports running pipelines in the cloud
+                </p>
+                <p>
+                  Raster Vision provides support for running pipelines using AWS Batch.
+                </p>
+              </div>
+              <div class="tech-spec text-white">
+                <img src="images/table-cells-large.svg" class="h-7 w-7"/>
+                <p class="font-semibold text-med">
+                  Supports configurable chip reading
+                </p>
+                <p>
+                  Geospatial images tend to be too large to feed directly into a neural network and must first be broken up into smaller “chips”.
+                </p>
+              </div>
+            </div>
           </div>
           <div class="text-center">
             <h2 class="text-white">
               Built on popular technologies & open source libraries including
             </h2>
-            <p class="lg-body text-white">
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit...
+            <p class="large-body text-white">
+              AWS (S3 and Batch), PyTorch, TorchVision, Albumentations, Rasterio, Shapely, Gdal and Numpy.
             </p>
             <div
               class="grid grid-cols-2 gap-2 sm:grid-cols-4 sm:gap-x-16 max-w-xl m-auto"
             >
-              <img src="images/placeholder.svg" />
-              <img src="images/placeholder.svg" />
-              <img src="images/placeholder.svg" />
-              <img src="images/placeholder.svg" />
+              <img src="images/logo-aws.svg" class="h-32 w-32"/>
+              <img src="images/logo-pytorch.svg" class="h-32 w-32"/>
+              <img src="images/logo-abulmentations.svg" class="h-32 w-32"/>
+              <img src="images/logo-numpy.svg" class="h-32 w-32"/>
```

Comment:
A few suggestions for the logos:
- I like the idea of having these all the same size, but because of the very different dimensions of the logos, this actually makes the logos that are closer to square look larger than others. Can you see if you can get them to look a bit closer in size? 
- `h-32` is too big for mobile screens, so we should add a smaller height for mobile screens.
- We should also link these logos to their respective libraries.